src/core/github.js

```diff
@@ -138,11 +121,42 @@ export async function run(conf) {
       showWarning(msg, name);
     }
   }
+  if (!conf.excludeGithubLinks) {
```

Comment:
Should we also exclude editor draft URL?

Reply:
Still under discussion... eventually, but it's up to the working group and requires a culture change at the W3C.

Reply:
Or do you mean when this option is added?

Reply:
Yes, when this option is added

Reply:
After the 2021 stuff, I'm going to remove conf.excludeGithubLinks, as this module shouldn't be affecting what goes in the headers.